crates/bitwarden-wasm-internal/src/init.rs

```diff
@@ -12,26 +19,35 @@ pub enum LogLevel {
 
 fn convert_level(level: LogLevel) -> Level {
     match level {
-        LogLevel::Trace => Level::Trace,
-        LogLevel::Debug => Level::Debug,
-        LogLevel::Info => Level::Info,
-        LogLevel::Warn => Level::Warn,
-        LogLevel::Error => Level::Error,
+        LogLevel::Trace => Level::TRACE,
+        LogLevel::Debug => Level::DEBUG,
+        LogLevel::Info => Level::INFO,
+        LogLevel::Warn => Level::WARN,
+        LogLevel::Error => Level::ERROR,
     }
 }
 
-#[wasm_bindgen]
-pub fn set_log_level(level: LogLevel) {
-    let log_level = convert_level(level);
-    set_max_level(log_level.to_level_filter());
-}
-
 #[allow(missing_docs)]
 #[wasm_bindgen]
 pub fn init_sdk(log_level: Option<LogLevel>) {
     console_error_panic_hook::set_once();
+
     let log_level = convert_level(log_level.unwrap_or(LogLevel::Info));
-    if let Err(_e) = console_log::init_with_level(log_level) {
-        set_max_level(log_level.to_level_filter())
-    }
+
+    let filter = EnvFilter::builder()
+        .with_default_directive(log_level.into())
+        .from_env_lossy();
+
+    let fmt = tracing_subscriber::fmt::layer()
+        .with_ansi(false) // Only partially supported across browsers
+        .with_timer(UtcTime::rfc_3339())
+        .with_writer(MakeWebConsoleWriter::new()); // write events to the console
```

Comment:
would like to manually validate this (I'm assuming tests in CI don't provide a way to confirm). If someone could recommend a way to validate it, I will do that.

Reply:
noting that @quexten shared some potential steps to manually validate this. I'll work on that